File: KosegenMatris.java
import java.util.Scanner;

public class KosegenMatris 
{
    public static void main(String[] args) 
    {
        Scanner girdi = new Scanner(System.in);
        System.out.print("Matris boyutunu giriniz = ");
        int a = girdi.nextInt();
        for (int i = 0; i < a; i++) 
        {
            for (int j = 0; j < a; j++) 
            {
                if (i == j) 
                {
                    System.out.print("1");
                } 
                else 
                {
                    System.out.print("0");
                }
            }
            System.out.println("");
        }
    }
}

File: KosegenMatris.py

"""
Diagonu 1 olan kare matris
"""
import numpy as np

girdi = input("Bir sayi giriniz : ")

girdi = int(girdi)

liste=np.zeros((girdi,girdi))

liste1=liste.astype(int)

for i in range(girdi):
    
    liste1[i,i] = 1

for i in range(girdi):
    
    for j in range(girdi):
        
        print(liste1[i,j],end = "   ")
        
    print("")
